cmake/Modules/FindDPDK.cmake

```diff
@@ -0,0 +1,59 @@
+# - Try to find DPDK include dirs and libraries
+#
+# Usage of this module as follows:
+#
+#     find_package(DPDK)
+#
+# Variables defined by this module:
+#
+#  DPDK_FOUND                System has DPDK include and library dirs found
+#  DPDK_INCLUDE_DIRS         The DPDK include directories.
+#  DPDK_LIBRARIES            The DPDK library
+
+find_package(Threads)
+find_package(NUMA)
+
+find_library(DPDK_LIBRARIES dpdk
```

Comment:
I think that this is incorrect, this find_library function only adds libdpdk to build but DPDK have 20+ other libraries that must be added (most of them with -Wl whole_archive, and some system libraries too) to compile. I can send you an  FindDPDK file that I use for projects as an example

Reply:
I had looked at the original makefile and saw a bunch of the libs, but it does seem that libsdpdk.a depends on pretty much everything else. So it seems like just pulling that in should be sufficient.